src/carriers/zfp_portmonitor/README.md

```diff
@@ -0,0 +1,11 @@
+
+zfp_portmonitor plugin 
+======================================================================
+Portmonitor plugin for compression and decompression of depth images using zfp library. Working on my machine, to include in yarp!
```

Comment:
I think the last sentence that starts with "Working on my machine" can be removed, but KUDOS for including a a README.